.github/workflows/stale.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Identify and close stale issues and PRs'
```

Comment:
@petersonjdNIH we probably want to disable this one, unless you want the robots closing issues after only a month.